webapp/channels/src/components/profile_popover/profile_popover_custom_attributes.tsx

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useEffect, useState} from 'react';
+import {useDispatch, useSelector} from 'react-redux';
+
+import {getCustomProfileAttributeFields} from 'mattermost-redux/actions/general';
+import {Client4} from 'mattermost-redux/client';
+import {getCustomProfileAttributes} from 'mattermost-redux/selectors/entities/general';
+
+import type {GlobalState} from 'types/store';
+
+type Props = {
+    userID: string;
+}
+const ProfilePopoverCustomAttributes = ({
+    userID,
+}: Props) => {
+    const dispatch = useDispatch();
+    const [customAttributeValues, setCustomAttributeValues] = useState<Record<string, string>>({});
+    const customProfileAttributeFields = useSelector((state: GlobalState) => getCustomProfileAttributes(state));
+
+    useEffect(() => {
+        const fetchValues = async () => {
+            const response = await Client4.getUserCustomProfileAttributesValues(userID);
+            setCustomAttributeValues(response);
+        };
+        dispatch(getCustomProfileAttributeFields());
+        fetchValues();
+    }, [userID, dispatch]);
+    const attributeSections = Object.values(customProfileAttributeFields).map((attribute) => {
+        const value = customAttributeValues[attribute.id];
+        if (!value) {
+            return null;
+        }
+        return (
+            <div
+                key={'customAttribute_' + attribute.id}
+                className='user-popover__custom_attributes'
+            >
+                <strong
+                    id='user-popover__custom_attributes-title'
```

Comment:
nit: `id` should be unique in a document

Reply:
Fixed this one